ext/view/theme.php

```diff
@@ -85,7 +85,17 @@ protected function build_info(Image $image, $editor_parts): string
 					<table style='width: 500px; max-width: 100%;' class='image_info form'>
 		";
         foreach ($editor_parts as $part) {
-            $html .= $part;
+            // This could be using something like MicroHTML? but the contents need to allow HTML anyway.
```

Comment:
You can add unsafe generic HTML into the middle of an otherwise-safe structure with rawHTML like

```
use function MicroHTML\rawHTML;
[...]
TH(rawHTML($part->header))
```

This also makes it easy to search for `rawHTML`, so if we see some invalid HTML being generated somewhere, we can focus on those parts :)

Reply:
That is nice. Will add this to my TODO list :)